webpack/components/Table/MainTable.js

```diff
@@ -54,6 +55,7 @@ const MainTable = ({
         body={emptyContentBody}
         happy={happyEmptyContent}
         search={!happyEmptyContent}
+        {...emptyStateProps}
```

Comment:
Could we remove the above line where `emptyStateProps` is set and call here directly `{...extraTableProps}` ?